app/support/reservations/validations.rb

```diff
@@ -23,7 +23,7 @@ module Reservations::Validations
     validate :duration_is_interval
 
     validates :category,
-              inclusion: { in: -> (r) { r.class::CATEGORIES }, allow_blank: false },
+              inclusion: { in: -> (r) { r.class::CATEGORIES }, allow_blank: true },
```

Comment:
I'd check with @drench on this. I'm pretty sure we need `allow_blank` to be false for instrument not available reservations.